website/docs/r/resiliencehub_app.html.markdown

```diff
@@ -0,0 +1,239 @@
+---
+subcategory: "Resilience Hub"
+layout: "aws"
+page_title: "AWS: aws_resiliencehub_app"
+description: |-
+  Terraform resource for managing an AWS Resilience Hub App.
+---
+
+# Resource: aws_resiliencehub_app
+
+Terraform resource for managing an AWS Resilience Hub App.
+
+## Example Usage
+
+### Basic Usage
+
+```terraform
+resource "aws_resiliencehub_app" "example" {
+  name                    = "example-app"
+  app_assessment_schedule = "Disabled"
+
+  app_template {
+    version = "2.0"
+
+    app_component {
+      name           = "appcommon"
+      type           = "AWS::ResilienceHub::AppCommonAppComponent"
+      resource_names = []
+    }
+  }
+}
+```
+
+### Complete Usage with Resources and Terraform Source
+
+```terraform
```

Comment:
The resource that we're documenting should be the first resource in the example.